src/qiboml/models/decoding.py

```diff
@@ -19,6 +20,8 @@ class QuantumDecoding:
     Args:
         nqubits (int): total number of qubits.
         qubits (tuple[int], optional): set of qubits it acts on, by default ``range(nqubits)``.
+            Optionally, the name of the wires to run on can be passed through this argument.
```

Comment:
Maybe I would add a small example to explain how to pass the argument in case we also want to set the `wire_names`.